spec/numberformat.html

```diff
@@ -77,18 +77,21 @@ <h1>
         1. Let _style_ be _numberFormat_.[[Style]].
         1. If _style_ is *"currency"*, then
           1. Let _currency_ be _numberFormat_.[[Currency]].
-          1. Let _cDigits_ be CurrencyDigits(_currency_).
+          1. Let _currencyPrecision_ be _numberFormat_.[[CurrencyPrecision]].
+          1. Let _cDigits_ be CurrencyDigits(_currency_, _currencyPrecision_).
+          1. Let _defaultRoundingIncrement_ be CurrencyRoundingIncrement(_currency_, _currencyPrecision_).
```

Comment:
Oh that's a good point. Great catch. I should have noticed that.

Reply:
oh, very good catch!

Reply:
A couple of thoughts:

1. It seems reasonable to throw if `currencyPrecision: "cash"` is set and `notation` isn't "standard", since cash rounding only really makes sense in standard notation.
2. If cash rounding only makes sense if `notation` is "standard", this suggests that the non-cash option for `currencyPrecision` should be something other than "financial" &mdash; maybe just "standard"?

Reply:
It's an issue not only when `notation` isn't "standard" but also when a different value was set for minimumFractionDigits or maximumFractionDigits. So the condition when this is used should be exactly the condition when `_cDigits_` is used.

I'm kind-of okay with any of the following outcomes:

1. Do a complicated check that the setting isn't present with conflicting options
2. Ignore the setting if conflicting options are present (this is how we behave when e.g. currencySign is present but style is not currency)
3. Define the algorithm similar to how you have it defined here and let people shoot themselves in the foot

Reply:
gut feelings when I see those potential outcomes worded that way:

3 seems bad (needlessly confusing for users)
1 seems less bad but not in keeping with the rest of the spec 
2 seems least bad

Reply:
Concretely the change would be the following in SetNumberFormatDigitOptions:

Undo your change on this line:

```diff
        1. Let _roundingIncrement_ be ? GetNumberOption(_options_, *"roundingIncrement"*, 1, 5000, 1).
```

And add the following line:

```diff
             1. Set _intlObj_.[[MinimumFractionDigits]] to _mnfdDefault_.
             1. Set _intlObj_.[[MaximumFractionDigits]] to _mxfdDefault_.
+            1. Set _intlObj_.[[RoundingIncrement]] to _defaultRoundingIncrement_.
```

I _think_ that gets the job done?